files/checks/check_systemd_service.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/bin/bash
```
The extra space breaks stuff.